include/ibamr/IBFEMethod.h

```diff
@@ -269,6 +269,13 @@ namespace IBAMR
  *   before the first time step if set to <code>TRUE</code>. Defaults to
  *   <code>false</code>.</li>
  * </ol>
+ *
+ * <h2>Handling Restart Data</h2>
+ * The caching of the IBFE restart data is not managed by SAMRAI's RestartManager. It 
+ * is instead handled by IBFEMethod::writeFEDataToRestartFile given a restart_dump_dirname
```

Comment:
```suggestion
 * is instead handled by IBFEMethod::writeFEDataToRestartFile() given a restart_dump_dirname
```
(doxygen needs the `()`s to create the link correctly)